test/fake_modules/prettier/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "my-cool-library",
+  "version": "9000.0.1",
```

Comment:
It’s over 9000!!!!

😂

Reply:
It's a copy of the given example on the prettier page… :wink: